.github/workflows/ci.yml

```diff
@@ -50,6 +55,19 @@ jobs:
       - name: Print Runner IP
         run: echo $MAIN_IP
 
+      - name: Generate TLS Certs
+        run: ./.github/workflows/scripts/tls.sh
+        env:
+          MAIN_IP: ${{ env.MAIN_IP }}
```

Comment:
This generates the certificates for CI usage. MAIN_IP is required because BindPlane's server certificate is created with IP SANS.